packages/mongo/oplog_v2_converter.js

```diff
@@ -24,30 +36,64 @@ on mongo 4
 const isArrayOperator = possibleArrayOperator => {
   if (!Object.keys(possibleArrayOperator).length) return false;
 
-  return !Object.keys(possibleArrayOperator).find(key => key !== 'a' && !key.match(/u\d+/));
+  return !Object.keys(possibleArrayOperator).find(
+    key => key !== 'a' && !key.match(/u\d+/)
+  );
 };
-const nestedOplogEntryParsers = (
-  { i = {}, u = {}, d = {}, ...sFields },
-  prefixKey = ''
-) => {
+
+function logOplogEntryError(oplogEntry, prefixKey, key) {
```

Comment:
Shouldn't you check here if `OPLOG_CONVERTER_DEBUG` is enabled?

Reply:
In case of an unrecoverable error, I've left this message. It's a different case, that var will suppress debug info, which is really verbose; this one, in contrast should never show.